images/cfw/opt/x1plus/lib/python/x1plus/utils.py

```diff
@@ -16,3 +16,26 @@ def serial_number():
     return subprocess.check_output(["bbl_3dpsn"], stderr=subprocess.DEVNULL).decode(
         "utf-8"
     )
+
+
+def get_MAC() -> str:
+    """Return the MAC address of the printer."""
+    if is_emulating():
+        return "CC:BD:D3:00:3B:D5"
+    # Get the device id of the currently-active network device.
+    devices = subprocess.Popen(["ip", "link"], stdout=subprocess.PIPE)
```

Comment:
You should always use `wlan0` for this.

Reply:
That was in the next PR, so I pulled it forward.